nodejs/week1/home/avg.js

```diff
@@ -0,0 +1,19 @@
+let sum = 0;
+
+if (process.argv.length === 2) {
+    console.log("Please enter a number");
+    return 0;
```

Comment:
In cases where our application does not succeed, we should not return zero, as zero is the standard number indicating success. Instead use a number like 1.

And since we want this to be returned to the user's command line as the exit code, use `process.exit()` instead of `return`.